app/src/main/java/co/nimblehq/compose/crypto/ui/screens/detail/DetailScreen.kt

```diff
@@ -87,10 +116,8 @@ fun DetailScreen() {
                         top.linkTo(currentPrice.bottom)
                         linkTo(start = parent.start, end = parent.end)
                     },
-                // TODO: Remove dummy image when work on Integrate.
-                priceChangePercent = "6.21",
-                displayForDetailPage = true,
-                isPositiveNumber = true
+                priceChangePercent = coinDetailUiModel.priceChangePercentage24hInCurrency,
```

Comment:
Could we round the value to 2 decimal places?

<img src="https://user-images.githubusercontent.com/6950766/192477984-84d883ff-3952-49a3-92cd-3a9e1bb7ce5a.png" width=300 />